### PR TITLE
show 3-get-api complete

### PR DESCRIPTION
**Pull request by geaon**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
 const express = require("express");
 const app = express();
-const { getTopics } = require("./controllers/topic-controller");
+const { getTopics } = require("./controllers/controllers");
+const endpoints = require("./endpoints.json");
 
 app.use(express.json());
```
Comment (nc-nick-jones) [inline app.js:6]: don't need express.json() yet.  please remove

File: app.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
 const express = require("express");
 const app = express();
-const { getTopics } = require("./controllers/topic-controller");
+const { getTopics } = require("./controllers/controllers");
+const endpoints = require("./endpoints.json");
```
Comment (nc-nick-jones) [inline app.js:4]: would be nice to require in endpoints in the controller and send.

**Review by nc-nick-jones — COMMENTED** [review]
this is okay! It'd be better practice to send your response in your controller for you endpoints though.  Keep up the good work!

File: app.js (hunk 2 above)
Comment (geaon) [reply app.js:4]: Hi Nick, I'm not 100% sure what you mean!

**Pull request merged**